wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -13,6 +13,7 @@ const Lobby = ({location}) => {
     const [users, setUsers] = useState([]);
     const [goSwipe, setGoSwipe] = useState(false);
     const [topTenMovies, setTopTenMovies] = useState(null);
+    console.log(location);
```

Comment:
nitpick but I think this console.log could potentially be confusing in the future when trying to debug. Would it be possible to remove this or reformat it to make it's purpose more clear?

Reply:
Ah for sure! Actually just forgot to take this one out. Good catch!